ci/Jenkinsfile.linux

```diff
@@ -50,25 +53,39 @@ pipeline {
       steps { sh 'make deps' }
     }
 
-    stage('status-go') {
-      steps { sh 'make status-go' }
-    }
-
-    stage('Client') {
-      steps { sh 'make nim_status_client' }
+    stage('Parallel Builds') {
```

Comment:
I don't know much about Jenkins, so asking as a way to learn:  how can the builds run in parallel if `nim_status_client` depends on `libstatus.a`?

Reply:
Ah fair, then that change makes no sense.